.rubocop.yml

```diff
@@ -141,6 +141,11 @@ Layout/ClassStructure:
 
 # RSpec -----------------------------------------------------------------------
 
+RSpec/VariableName:
```

Comment:
Disabled to pass correct headers name in request
e.g.(x_refresh_token could not be convert to X-Refresh-Token)
Will change it if some one has propositions